src/tests/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+
+if(ENABLE_TESTING)
+  foreach(PROG test_elements )
```

Comment:
I am not sure, why you are doing this in a loop, but I might make it easier to add more tests later.

Reply:
Yes, that was my reasoning.